scripts/qesap/test/unit/test_subprocess_run.py

```diff
@@ -29,10 +29,10 @@ def test_multilines():
     test_text = ''
     for s in str_list:
         test_text += (s*10) + "\n"
-    exit_code, stdout_list = subprocess_run(['echo', test_text])
+    _, stdout_list = subprocess_run(['echo', test_text])
     assert len(stdout_list) == len(str_list)+1
-    for i in range(len(str_list)):
-        assert str_list[i]*10 == stdout_list[i].strip()
+    for _, x in enumerate(str_list):
```

Comment:
like

Reply:
https://github.com/SUSE/qe-sap-deployment/runs/8129579939?check_suite_focus=true

This change result in a unit test failure
```
          for _, x in enumerate(str_list):
  >           assert x*10 == x.strip()
  E           AssertionError: assert 'aaaaaaaaaa' == 'a'
  E             - a
  E             + aaaaaaaaaa
```